websites/W/watchmovieshd/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Toxicity",
+		"id": "794558195921846292"
+	},
+	"service": "watchmovieshd",
+	"description": {
+		"en": "Watch movies that get updated daily online in hd quality."
+	},
+	"url": "watchmovieshd.ru",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/iRop2fh.png",
```

Comment:
The website's logo is not a solid black square. Use the actual logo while ensuring that the resolution meets guidelines.

Reply:
it isnt a black square that logo have a transparent background

Reply:
<details>
<summary>This is what I see</summary>
<img width="960" alt="image" src="https://user-images.githubusercontent.com/32113157/189771923-af4ea6b6-8eb6-42c6-8015-446d2918f79b.png">
<p>No transparency to be seen, double check the image?</p>
</details>

Reply:
Can confirme it's a black square

Reply:
i think i just resolve it